### ISSUE TITLE
Add-on bar reappearing

### ISSUE THREAD

**Issue by sesquipedality** [AllSeeingEye/Barlesque#18]
Version 1.15

I think that the patch to update the bar when there are new notifications is causing problems with some extensions. The one I believe I'm having difficulty with is Linkification, which updates its icon whenever you change web pages, which results in the add-on bar popping up every time I load a new page or change a tab.

My current workaround is to set auto-hide to 1s, so it goes away again.

Suggested fixes:

(a) (OK, but not the best) an option to turn off automatic unhide.
(b) (Better but probably much harder) Ability to configure the automatic unhide function on a per icon basis - ie don't unhide if these icon's change, do unhide if these other ones do.

Thanks,

Christi

**Comment by alphapapa** [comment]
Yes, I'm seeing this also.  In fact, it not only happens when loading a page or changing a tab, but on some pages it happens every few seconds, period.  For example, when I was writing a long email in a Gmail tab, about every 10-15 seconds, there would be some hard disk activity for a second, and the addon bar would appear again.

I have these extensions showing in the bar, but I'm not sure which, if any, of them would cause this:

Stylish
Cookie Monster
NetVideoHunter
Lazarus

**Comment by cbarrick** [comment]
Firebug causes this too. Maybe you could maintain an internal blacklist for addons that cause this issue and ignore when they update their icon.
